### ISSUE TITLE
IOTA sends illegal entity type to CB (may also happend with entity id)

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/iotagent-node-lib#1124]
We have found that under some circustances (@danielvillalbamota please add the detail to this issue) IOTA sends this `POST /v2/op/update` to CB, with an illegal `type`

```
{
	"actionType": "append",
	"entities": [{
		"id": "GW02",
		"type": {
			"value": "2021-10-28T09:25:23.000",
			"type": "string"
		},
		"temperature": {
			"type": "Number",
			"value": "20",
			"metadata": {
				"TimeInstant": {
					"type": "DateTime",
					"value": "2021-10-28T07:57:06.706Z"
				}
			}
		},
		"TimeInstant": {
			"type": "DateTime",
			"value": "2021-10-28T07:57:06.706Z"
		}
	}, {
		"id": "HN01",
		"type": "heritageNode",
		"dateObserved": {
			"type": "DateTime",
			"value": "dateObs.type",
			"metadata": {
				"TimeInstant": {
					"type": "DateTime",
					"value": "2021-10-28T07:57:06.706Z"
				}
			}
		},
		"TimeInstant": {
			"type": "DateTime",
			"value": "2021-10-28T07:57:06.706Z"
		}
	}]
}
```

IOTA should avoid this, not only for `type` but also for `id`

**Comment by fgalan** [comment]
Fixed by PR #1130 

Should this issue be closed? Or should be test before by @danielvillalbamota in the environment where originally happened?

**Comment by fgalan** [comment]
> Fixed by PR #1130

I has been revered by PR https://github.com/telefonicaid/iotagent-node-lib/pull/1136
